x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ccr/AutoFollowMetadata.java

```diff
@@ -215,6 +220,26 @@ public AutoFollowPattern(String remoteCluster,
             this.remoteCluster = remoteCluster;
             this.leaderIndexPatterns = leaderIndexPatterns;
             this.followIndexPattern = followIndexPattern;
+            this.active = active;
+        }
+
+        // for testing purpose
+        public AutoFollowPattern(String remoteCluster,
```

Comment:
It looks like this was only added for avoiding changing call sites but I'd rather we didn't. You can use IntelliJ refactoring (change signature on this method to add the `boolean active` parameter as the fourth argument setting the default value to true, and ignore the conflict with the other override) to automatically handle the call sites (although you might have to worry about line-wrapping). Then we can avoid adding this method.

Reply:
Thanks Jason for spotting this, it should not have been committed. I added this ctor to avoid some work and planned to come back to it later, but didn't :man_facepalming: 

I pushed 05f7d2416bda517d33ba6bcc22514f93e97b05b1